src/resolvers/fsresolver.ts

```diff
@@ -1,15 +1,23 @@
 import * as fs from "fs";
 import * as path from "path";
-import { promisify } from "util";
 import { SubResolver } from ".";
 import { ResolverContext } from "./subresolver";
 
-const statAsync = promisify(fs.stat);
+const statAsync = (path: string): Promise<fs.Stats> =>
```

Comment:
Fs resolver is used only in node context - same as stat, promisify can stay.

Reply:
ATM, front-end compiler complains :(

Reply:
It really shouldn't compile node stuff

Reply:
Can I readdress that in a future merge?

Reply:
👍